python_backend/api/v1/endpoints/profile.py

```
@@ -0,0 +1,166 @@
+                "sessionEmotionLevel": 5,
+                "sessionDirectnessLevel": 3,
+                "responses": {"greeting": "Hello!"},
+                "completedAt": "2025-08-10T00:00:00Z"
+            }
+        }
+
+# Dependency injection
+def get_database_storage():
+    return DatabaseStorage()
+
+def get_user_preferences_service(db: DatabaseStorage = Depends(get_database_storage)):
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)